pom.xml

```diff
@@ -4,13 +4,14 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.401</version>
+        <version>2.21</version>
     </parent>
 
     <artifactId>versioncolumn</artifactId>
     <packaging>hpi</packaging>
-    <version>0.3-SNAPSHOT</version>
-    <name>Jenkins Version Column plugin</name>
+    <version>2.0-SNAPSHOT</version>
```

Comment:
bumped the major version to clarify/express the core requirement bump and new features (though 100% compatible)